iacrcc/iacrdoc.tex

```diff
@@ -392,18 +388,28 @@ \subsection{Abstract}
 \texttt{abstract} environment as usual.  Note that the keywords should
 be given before starting the \texttt{abstract} environment.
 
-For \texttt{final} versions of papers, an additional text-only
+For \texttt{final} versions of papers, an additional ``text-only''
 abstract is required. This abstract is contained in the
-\texttt{textabstract} environment, and should be just
-plain text (i.e., it should not contain macros).
+\texttt{textabstract} environment, and should not contain
+user-defined macros.
 It will be used for indexing and production of
 {HTML} pages to describe the paper. As such, it is just as important
 as the classical \texttt{abstract} of a paper because it contains a
 textual summary that readers will use to decide if the paper is worth
 reading. The only difference is that the contents of the
-\texttt{textabstract} is constrained on what it may contain.  The
-contents of this environment will be written to a file that ends with
-\texttt{.abstract} when you compile your \LaTeX, but will not be
+\texttt{textabstract} is constrained on what it may contain.
+
+You may use unicode such as in \verb+Paul Erdős+ or
+diacriticals like \verb+F\"ur Elise+. You may also use (inline)
+mathematics in the \texttt{textabstract} environment as well as
+(for example) the itemize environment. Use-defined macros are
+\emph{not} allowed. We do not have a complete list of allowed
+\LaTeX (which can be succesfully converted to HTML) but when
```

Comment:
Unfortunately they won't be able to test it until their paper is accepted and they try to upload their final version. For this reason I'd say
> but you will find out when you upload your final version.

Reply:
Good point. Will adjust the text.